SU-HSDO/suhumsci
# SHS 5130 Use <hr> when there is no color band text

# READY FOR REVIEW

## Summary
This PR checks the Color Band component for text, and, if there is no text, prints an <hr> tag instead of an empty heading tag. 

## Urgency
medium

## Steps to Test
1. Edit a page
2. Add a new page component, and choose Color Band
3. Do not add text to the text field and save
4. Ensure that if the color band has no text, an `<hr>` tag prints inside the color band. 
5. Ensure that color bands with text still print the same as before.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
